apps/opik-backend/src/main/java/com/comet/opik/api/Alert.java

```diff
@@ -0,0 +1,70 @@
+package com.comet.opik.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonView;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+@Builder(toBuilder = true)
+@JsonDeserialize(builder = Alert.AlertBuilder.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public record Alert(
+        @JsonView( {
+                Alert.View.Public.class, Alert.View.Write.class}) UUID id,
+
+        @JsonView({Alert.View.Public.class,
+                Alert.View.Write.class}) @NotBlank @Size(max = 255) String name,
+
+        @JsonView({Alert.View.Public.class,
+                Alert.View.Write.class}) Boolean enabled,
+
+        @JsonView({
+                Alert.View.Public.class, Alert.View.Write.class}) @NotNull UUID webhookId,
```

Comment:
If I'm not mistaken, we are not exposing webhookId directly, so we should define a webhook object and embed it here.